### PR TITLE
Update sessions.rst

### PR DESCRIPTION
**Pull request by ThomasLandauer**
First step to explain how to activate Namespaced Attributes, as requested by https://github.com/symfony/symfony-docs/issues/7378

### CODE REVIEW

File: components/http_foundation/sessions.rst [hunk 1]
```diff
@@ -185,25 +170,27 @@ and remember me login settings or other user based state information.
...
+    Returns ``true`` if the attribute exists.
 
 :method:`Symfony\\Component\\HttpFoundation\\Session\\Attribute\\AttributeBagInterface::replace`
-    Sets multiple attributes at once: takes a keyed array and sets each key => value pair.
+    Sets multiple attributes at once using an associative array (``name => value``).
+    If the attributes existed, they are replaced; if not, they are created.
 
 :method:`Symfony\\Component\\HttpFoundation\\Session\\Attribute\\AttributeBagInterface::remove`
-    Deletes an attribute by key.
+    Deletes an attribute by name.
```
Comment (HeahDude) [inline components/http_foundation/sessions.rst]: and returns its value.

**Review by HeahDude — COMMENTED** [review]
One minor comment, nice PR, thanks!

**Review by HeahDude — APPROVED** [review]
👍

**Comment by drupol** [comment]
Hi,

I would rather use a valid syntax example for `config.yml`:

```
services:
    session:
      class: Symfony\Component\HttpFoundation\Session\Session
      arguments: ["@session.storage", "@session.namespacedattributebag", "@session.flash_bag"]

    session.namespacedattributebag:
      class: Symfony\Component\HttpFoundation\Session\Attribute\NamespacedAttributeBag
```

**Comment by ThomasLandauer** [comment]
Sorry, don't know what you mean - can't remember all details half a year later....

Do you want to *add* something, or do you want me to *change* something?

File: components/http_foundation/sessions.rst [hunk 2]
```diff
@@ -205,35 +219,22 @@ the array::
...
-:method:`Symfony\\Component\\HttpFoundation\\Session\\Attribute\\AttributeBagInterface::clear`
-    Clear the bag.
+To activate namespaced attributes, add this to your ``services.yml``::
+
+    # app/config/services.yml
+    services:
+        session:
+            class: Symfony\Component\HttpFoundation\Session\Session
+            arguments:
+                - @session.storage
```
Comment (mhabibi) [inline components/http_foundation/sessions.rst]: shouldn't it be @session.storage.native?

Comment (javiereguiluz) [reply components/http_foundation/sessions.rst]: @HeahDude you approved this PR ... but I'd like to ask you what do you think about this comment from @mhabibi. Do you agree with it? Thanks!

Comment (HeahDude) [reply components/http_foundation/sessions.rst]: We don't care about the implementation here, so we don't need to be specific in that case and the alias is used. From Symfony 3.3, it should `SessionStorageInterface` instead (autowired by default). See https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Resources/config/session.xml#L21, that keeps BC with the old alias.
We can merge as is, thanks!

**Comment by javiereguiluz** [comment]
@ThomasLandauer sorry it took us so long to merge this. I like a lot your improvements. I only removed some example that showed how to configure this on Symfony. Since this is the stand-alone component doc, we never show how to integrate it with the whole framework. Thanks!

**Comment by pbowyer** [comment]
@javiereguiluz Can the examples that showed how to configure Symfony be added to the Symfony session documentation? It took ma a while to find this PR to confirm how to use `NamespacedAttributeBag` with Symfony 4.

**Comment by javiereguiluz** [comment]
@pbowyer I've created #10494 to not forget about this. Thanks!

**Pull request closed without merging**